src/components/Filter/Filter.js

```diff
@@ -0,0 +1,314 @@
+import React, { useState, useEffect } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faXmark, faMinus } from '@fortawesome/free-solid-svg-icons';
+import RangeSlider from '../RangeSlider/RangeSlider';
+import {
+  StyleFilter,
+  H2,
+  GraySpan,
+  GraySmallSpan,
+  TitleSpan,
+  Header,
+  HeaderDiv,
+  CloseBtn,
+  BoldSpan,
+  Body,
+  BodyDiv,
+  BodyInner,
+  BodyContainer,
+  BodyWrapper,
+  BodyWrapperDiv,
+  FeeDiv,
+  FeeInput,
+  BodyTypeDiv,
+  BodyWrapperInner,
+  BodyTypeFilter,
+  CheckboxInput,
+  BodyTypeFilterInner,
+  Footer,
+  BlackBtn,
+  MoreSpan,
+  MoreDiv,
+  AllCancelDiv,
+  AllCancelSpan,
+  FooterDiv,
+  RangeBox,
+  RangeBoxInner,
+  Slider,
+  RangeDiv,
+} from './StyleFilter';
```

Comment:
```suggestion
import * as S from './StyleFilter';
```
스타일 컴포넌트를 불러와서 사용하는 방법은 좋지만, 너무 길어져서 가독성에 오히려 문제를 주고 있습니다.
이럴 경우 위와 같이 수정 한뒤, 아래 뷰 부분에서 스타일 컴포넌트 앞에 S.을 붙여 주면 동일하게 사용 가능합니다!